packages/form-js-editor/src/features/properties-panel/entries/ValueEntry.js

```diff
@@ -99,6 +99,6 @@ function Value(props) {
     id,
     label: 'Value',
     setValue,
-    validate: validateFactory(getValue())
+    validate: validateFactory(getValue(), (entry) => entry.value)
```

Comment:
I know it's existing behavior, but with this validation, our users can't specify an empty value option. I'm just thinking that might be a valid use case, actually 🤔

Reply:
Maybe if it comes up as a request, we can look into it. But I think for now let's ignore it's too minor to care IMO.